EssentialCSharp.Web/Views/Home/Home.cshtml

```diff
@@ -1,5 +1,13 @@
 <div class="py-2 px-2"> 
-    <div> 
+    <div>
+       @*  @using (Html.BeginForm())
+        {
+            <script src="https://js.hcaptcha.com/1/api.js" async defer></script>
+            <div class="form-group">
+                <div class="h-captcha" data-sitekey="b9235f58-3d8d-4394-ab8e-78b35a6d69c5"></div>
+            </div>
+            <button type="submit" class="btn btn-primary">Submit</button>
+        } *@
```

Comment:
Don't we need this to make it work?

Reply:
As per the other thread, we are going to merge this and just holding this code as a reference.